client/client-lib/src/lib.rs

```diff
@@ -288,6 +288,13 @@ impl<T: AsRef<ClientConfig>> Client<T> {
         self.submit_tx_with_change(tx, batch, &mut rng).await
     }
 
+    /// Returns a bitcoin address suited to perform a fedimint [peg-in](Self::peg_in)
+    ///
+    /// This function requires a cryptographically secure randomness source, and utilizes the [wallet-clients](crate::wallet::WalletClient)
+    /// [get_new_pegin_address](crate::wallet::WalletClient::get_new_pegin_address) to **forge** a bitcoin-address
```

Comment:
"forge" has quite a specific meaning in cryptography and it's not what we are doing here. I'd say "to derive a bitcoin address from the federation's public descriptor" maybe adding "by tweaking it".